src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -1776,6 +1779,31 @@ namespace GraphQL.Subscription
         public System.Collections.Generic.IDictionary<string, System.IObservable<GraphQL.ExecutionResult>>? Streams { get; set; }
     }
 }
+namespace GraphQL.Transports.Json
+{
+    public class GraphQLRequest
+    {
+        public const string EXTENSIONS_KEY = "extensions";
+        public const string OPERATION_NAME_KEY = "operationName";
+        public const string QUERY_KEY = "query";
+        public const string VARIABLES_KEY = "variables";
```

Comment:
Perhaps. It seems to me that I used them somewhere in my projects. I will not upset if they are removed.

Reply:
Then I have removed them.  I feel they are specific to GraphQL-over-HTTP and as such are part of the JSON serializer, rather than a simple data model which can be used by a binary converter or any other converter.  We can make them public within the `GraphQLRequestJsonConverter` if you like.